### DISCUSSION TITLE
Multiple Container with same id

### DISCUSSION THREAD

**Discussion by jackguoAtJogg** [clauderic/dnd-kit#1153] [Q&A]
I wonder if there's a way that I could have multiple containers that can have the same id and it can be drag n drop but won't able to drop in container that have the same id.

**Comment by pstrassmann** [comment]
Each container needs a unique id. But you can include information in the `data` prop ([docs](https://docs.dndkit.com/api-documentation/droppable/usedroppable#data)) that can be used by your dnd event handlers (onDragOver, onDragEnd, etc) to decide whether to allow drops or not
